### DISCUSSION TITLE
Patching Plumed to Lammps-allegro

### DISCUSSION THREAD

**Discussion by tienmng** [mir-group/allegro#32] [General]
Hi,

Has anyone patched Plumed to lammps along with allegro? I would appreciate it very much if you could tell me how to do it. It would be great if you could provide your code.

Thanks!

**Comment by Linux-cpp-lisp** [comment]
Hi @tienmng ,

Thanks for your interest in our methods!

This thread on `pair_nequip` should hopefully be relevant: https://github.com/mir-group/pair_nequip/discussions/35
